src/renderer/components/ErrorView.jsx

```diff
@@ -8,28 +8,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {Grid, Row, Col} from 'react-bootstrap';
 
-import {SECOND} from 'common/utils/constants';
-
-//import {ipcRenderer} from 'electron';
-// import {OPEN_EXTERNAL} from 'common/communication';
-
 export default function ErrorView(props) {
     const classNames = ['container', 'ErrorView'];
     if (!props.active) {
         classNames.push('ErrorView-hidden');
     }
 
-    // function handleClick(event) {
-    //   event.preventDefault();
-    //   console.log('TODO: send to main');
-    //   ipcRenderer.send(OPEN_EXTERNAL, props.url);
-    // }
-
-    let retry = null;
-    if (props.retry) {
-        const seconds = (Date.now() - props.retry) / SECOND;
-        retry = <div className='retry-info'><p>{`Trying to load again in ${seconds}`}</p></div>;
-    }
```

Comment:
I was hoping to fix the timer as visually was wrong, but functionality was working ok. Any reason to remove it? it would prevent constant reloads of the server if it can't connect.

Reply:
Currently the new loading screen is showing instead of that "Cannot connect to Mattermost" screen. That screen now only shows if we have no connection. UX seems better, though we do provide the user with less information.

I would envision that we include that timer on the new loading screen if at all. @deanwhillier thoughts?

Reply:
we might not have to include the timer, but keep the retries at least? so it doesn't end in some sort of "eternal loading" screen

Reply:
Oh the retries are still happening behind the scenes, there's just no indication to the user.